src/pages/components/date-picker/style.mdx

```diff
@@ -1,14 +1,31 @@
 ---
 title: Date picker
 description:
-  Date and time pickers allow users to select a single or a range of dates and
-  times.
+  The following page documents visual specifications such as color, typography,
+  structure, and size.
 tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 ---
 
+<PageDescription>
+
+The following page documents visual specifications such as color, typography,
+structure, and size.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Color</AnchorLink>
+<AnchorLink>Typography</AnchorLink>
+<AnchorLink>Structure</AnchorLink>
+<AnchorLink>Size</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
 ## Color
 
-### Date input
+### Date picker color
 
```

Comment:
We currently are not including an additional H4 there since we have an H3 already to specify the section in the style template. If we find down the road that we may need to add an additional H4 for that section, we can implement those changes in the style template and on our style tab pages.